src/app/app.component.html

```diff
@@ -1,14 +1,37 @@
-<app-header></app-header>
+<app-header>
+  <span class="userName">Harry Potter</span>
+  <app-button [buttonText]="btnTextLogout"></app-button>
+</app-header>
 <div class="app-root">
   <!-- Search component -->
   <!--<app-search></app-search>-->
 
   <!-- Info component -->
-  <!--<app-info></app-info>-->
+  <app-info [title]="infoTitle" [text]="infoText">
+    <app-button [buttonText]="btnTextAddNewCourse"></app-button>
+  </app-info>
 
   <!-- Course Card component -->
-  <!--<app-course-card></app-course-card>-->
+  <app-course-card
+    [isEditabel]="true"
+    [course]="mockedCourse"
+    (clickOnShow)="showCourse($event)"
+    (clickOnDelete)="deleteCourse($event)"
+    (clickOnEdit)="editCourse($event)"
+  >
+    <app-button showCourseBtn [buttonText]="btnTextShowCourse"></app-button>
```

Comment:
Obviously, this is correct with content projection within CourseCardComponent, however it seem a little bit overcomplicated. I think it is safe and cleaner to declare the action buttons within the CourseCardComponent itself and render them on the corresponding condition.
Probably we will not have more action buttons for a given course other than show, delete and edit.

Reply:
I was confused a little bit, maybe I misunderstood something. 
The description said:
Course card
Requirements
...
**7. Should get button(s) from content projection (<ng-content></ng-content>).**

I will fix it.
Did I misunderstood, or the description wasn't so accurate?
I would like to know what I should pay more attention to in the future.